DasharoModulePkg/Library/DasharoVariablesLib/DasharoVariablesLib.c

```diff
@@ -511,8 +526,8 @@ DasharoMeasureVariables (
   EFI_STATUS  Status;
 
   Status = MeasureVariables (&gDasharoSystemFeaturesGuid);
-  if (Status == EFI_SUCCESS)
-    Status = MeasureVariables (&gApuConfigurationFormsetGuid);
+  Status |= MeasureVariables (&gApuConfigurationFormsetGuid);
+  Status |= MeasureVariables (&gSovereignBootWizardFormSetGuid);

```

Comment:
Do you really want to do bitwise OR on `EFI_STATUS`?  This can result in reporting status which doesn't correspond to any error.

Reply:
Hmm, right.

I just didn't want to skip measurements of one set of variables, just because the previous set failed for some reason.

Reply:
Added two status variables to handle it (amended commit): https://github.com/Dasharo/edk2/pull/256/commits/d761adb6e1fce9baf0e6811c852c0eae6b002397#diff-3508fed3399ece58e1791433a5621b402f866ce70f730469cfc97f33b52ac284R531